### ISSUE TITLE
Is AWS Polly premium only now?

### ISSUE THREAD

**Issue by barolo** [AwesomeTTS/awesometts-anki-addon#178]
I have my own keys and there’s nowhere to input them.

**Comment by lucwastiaux** [comment]
Yes, AWS Polly is only available on Premium, because the boto3 framework which AWS requires cannot be packaged into an Anki add-on. So the only way to make it work is to use an external server to forward access. If you want to use your own keys, I suggest using Google Cloud, Azure, Watson or Naver. most of these offer a free tier also.

**Comment by barolo** [comment]
I'm using Watson, the thing is as you're probably aware that AWS have way much higher free tier caps. [and better voices in general]
I don't fully grasp the boto issue, how addons like `Read Aloud` for chrome/firefox do it?

**Comment by lucwastiaux** [comment]
Services like Google, Azure, Naver, Watson provide a standard REST API which python can access without an issue. Amazon requires installation of their relatively large SDK for python, which is not possible inside an Anki addon. That's the current situation. I can try to take a look at how Read Aloud does it. You can also consider becoming a fan on patreon if you need AWS voices working right now (though it requires a $5/month subscription).

**Comment by lucwastiaux** [comment]
Looks like Read Aloud embedded a minified AWS SDK: https://github.com/ken107/read-aloud/blob/master/js/aws-sdk.js
i'll have to look into whether something like that is possible for AwesomeTTS.

**Comment by luc-vocab** [comment]
HyperTTS lets you use your own key on AWS Polly. If that's something you need please migrate to HyperTTS.
